### PR TITLE
Additional Holodeck Simulations (Jungle, Rainy Cafe, Ocean, Holidays) + Additional Ambient Sounds

### PR DESCRIPTION
**Pull request by Laniakeaaaa**
### Adds Four Additional Simulations to the Holodeck System with ambiences.

- Jungle Biome Simulation
- Rainy Cafe
- Ocean
- Holiday Simulation - (Christmas Tree)

Adds two new rain ambient tracks from Freesound

### CODE REVIEW

File: maps/nerva/nerva_holodecks.dm [hunk 1]
```diff
@@ -80,4 +90,55 @@
...
+/turf/simulated/floor/holofloor/urist/jungle_water
+	name =  "murky water"
+	desc = "Holographic deep, dank water. You swear you can see something moving below occasionally."
+	base_desc = "Holographic deep, dank water. You swear you can see something moving below occasionally."
+	icon = 'icons/urist/jungle/turfs.dmi'
+	icon_state = "rivernew"
+
+/turf/simulated/floor/holofloor/urist/pavement
+	name = "Pavement"
+	desc = "Holographic deep, dank water. You swear you can see something moving below occasionally."
```
Comment (Glloyd) [inline maps/nerva/nerva_holodecks.dm:134]: i'm not sure this is a proper description for pavement

**Review by Glloyd — CHANGES_REQUESTED** [review]

File: maps/nerva/nerva_holodecks.dm (hunk 1 above)
Comment (Laniakeaaaa) [reply maps/nerva/nerva_holodecks.dm:134]: no more dank pavements

Comment (Laniakeaaaa) [reply maps/nerva/nerva_holodecks.dm]: Ooops, haha

**Comment by Laniakeaaaa** [comment]
Fixed.

**Review by Glloyd — APPROVED** [review]

**Pull request merged**